SU2_CFD/include/numerics_structure.hpp

```diff
@@ -4175,7 +4186,7 @@ class CFEAElasticity : public CNumerics {
 
   void Set_YoungModulus(unsigned short i_DV, su2double val_Young);
 
-  void SetElement_Properties(CElement *element_container, CConfig *config);
+  virtual void SetElement_Properties(CElement *element_container, CConfig *config);
```

Comment:
Why has this become virtual? Or was it already?

Reply:
Nice catch. I think I changed it at some point to overload the SetElement_Properties, but then I didn't need it anymore and I forgot to change it back. Solved.